torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2001,6 +2001,24 @@ def process_output(eigpair):
         ),
     ]
 
+
+def sample_inputs_einsum(op_info, device, dtype, requires_grad=False, **kwargs):
+    tensors = [
+        make_tensor((2, 2, 3), device, dtype, requires_grad=requires_grad),
+        make_tensor((2, 3), device, dtype, requires_grad=requires_grad),
+        make_tensor((2, 1, 3), device, dtype, requires_grad=requires_grad),
+    ]
+
+    # (<number of operands>, <equation>)
+    test_cases = [
```

Comment:
How should we think about reconciling the sample inputs define here with the many cases (which check autograd) defined in test_torch.py?

https://github.com/pytorch/pytorch/blob/a3a75bd35ec6e1ad7371511cb06c085d74fde6ba/test/test_linalg.py#L4227

Reply:
I picked some tests from test_linalg to use in common_methods_invocations and removed the gradcheck test from test_linalg.